app/validators/date.js

```diff
@@ -5,7 +5,7 @@
 
 import Ember from 'ember';
 import Base from 'ember-cp-validations/validators/base';
-const moment = self.requirejs.entries['moment'] && self.require('moment')['default'];
+const moment = (self.requirejs.entries['moment'] || self.requirejs.entries['moment/index']) && self.require('moment')['default'];
```

Comment:
We should add a nice `loaderjs.has` helper method for this, but that is another issue altogether :smiley_cat: